app/models/authenticator/ldap.rb

```diff
@@ -34,6 +34,9 @@ def find_or_create_by_ldap(username)
       user = userprincipal_for(username)
       return user unless user.nil?
 
+      user = autocreate_user(username)
+      return user unless user.nil?
```

Comment:
@furlongm I apologies for letting this review slip through the cracks on me.

I patched your change onto a test VM and I think this is a good simple change 👍 

I have a minor suggestion. The two `return user unless user.nil?` lines could be eliminated. 

Instead of:

> +      user = autocreate_user(username)
> +      return user unless user.nil?

Perhaps this would be better:

> +      user = autocreate_user(username) if user.nil?

Also please ensure this change does not break the spec:
https://github.com/ManageIQ/manageiq/blob/master/spec/models/authenticator/ldap_spec.rb

It would also be good to update the spec to exercise your this change.

Thank you, JoeV